features/git-prune-branches/feature_branch/behind_main/with_open_changes.feature

```diff
@@ -6,27 +6,49 @@ Feature: git prune-branches: don't remove the current empty feature branch if th
 
 
   Background:
-    Given I have a feature branch named "feature" behind main
-    And I have a feature branch named "stale_feature" behind main
-    And I am on the "feature" branch
+    Given the following commits exist in my repository
+      | BRANCH | LOCATION         | MESSAGE     | FILE NAME |
+      | main   | local and remote | main commit | main_file |
+    And I have a feature branch cut from "Initial commit" named "stale_feature_initial"
+    And I have a feature branch cut from "main commit" named "stale_feature_main"
```

Comment:
I'm hesitant to rename "cut from" to "tip at", since it must be clear where the branch is coming from in order to asses whether it is stale or not. I think in Git operations: this branch was cut from here, then there were no commits added to it. Just telling me "this branch has its tip here now" isn't really enough to describe a branch to me.